src/Diagrams/Swimunit/Dotmatrix.hs

```diff
@@ -0,0 +1,843 @@
+{-# LANGUAGE NoMonomorphismRestriction #-}
+
+module Diagrams.Swimunit.Dotmatrix
```

Comment:
Would `Diagrams.TwoD.DotMatrix` be more consistent with other modules names?

Also remember to add this to `diagrams-contrib.cabal`.